### ISSUE TITLE
WiFi RSSI signal strength reading/indicator

### ISSUE THREAD

**Issue by CryptoIceMLH** [bitaxeorg/ESP-Miner#319]
I Think it would be very useful both for support issues but also for user monitoring 
to modify the code to export the ESP32 wifi modules RSSI ( also known as the wifi signal strength )

The RSSI (Received Signal Strength Indicator) can be checked by the ESP32 to determine the WiFi connection strength between the ESP32 and the specific WiFi network you’re trying to connect to (e.g. home router or any access point). This can be useful in case you’re having some issues with the WiFi connection or keeps disconnecting sporadically or for some upcoming applications to monitor the max distance usable for a bitaxe away from the Access point. 
You can then check the RSSI for the network and judge if the WiFi signal power needs adjusting or if you simply need to place the ESP32 closer to the access point.

Generally speaking, the lower the RSSI value, the weaker the signal is, and vice versa. The range for RSSI value is from 0 down to -120 dBm. Having a 0 dBm is hard to achieve and it means you’ve got an amazingly strong connection. And low values from -90 down to -120 dBm are unusable at all
RSSI is an estimated measure of the WiFi signal strength for a specific network (router or access point). The return value has the following form and unit (-x dBm). Which means a lower absolute value indicates a more powerful connection. 

RSSI > -30 dBm | Amazing
RSSI < – 55 dBm | Very good signal
RSSI < – 67 dBm | Fairly Good
RSSI < – 70 dBm | Okay
RSSI < – 80 dBm | Not good
RSSI < – 90 dBm | Extremely weak signal (unusable)

WiFi.RSSI();  

ref doc: [ https://www.arduino.cc/reference/en/libraries/wifi/wifi.rssi/](url)

**Comment by CryptoIceMLH** [comment]
Nudging this again as a non urgent reminder !

**Comment by skot** [comment]
I agree, exposing the current RSSI to AxeOS would be really helpful.

Maybe even show it in the new SSID chooser @WantClue ?

**Comment by WantClue** [comment]
The RSSI has been added to the API endpoint

**Comment by CryptoIceMLH** [comment]
thanks
